plugins/env/app/views/samson_env/_fields.html.erb

```diff
@@ -1,9 +1,15 @@
 <fieldset>
   <legend>
-    Env variables written to the .env file or added to kubernetes before deploy
-    <%= additional_info SamsonEnv::HELP_TEXT %>
+    Environment Variables
   </legend>
 
+  <% if ENV['DEPLOYMENT_ENV_REPO'] %>
+    <%= form.input :use_env_repo, label: "Use GitHub repository to manage deployment environment variables",
```

Comment:
could say `Use #{repo} to manage environment variables`

Reply:
@bcolferzd ☝️